demo/static/sass/demo-ltr.scss

```diff
@@ -9,6 +9,11 @@
 // ------------------------------
 $layout-direction: ltr;
 
+// ------------------------------
+// #LIB
+// ------------------------------
+@import '../../../node_modules/bi-app-sass/bi-app/bi-app-ltr';
```

Comment:
In fact, that partial could set the $layout-direction too so that wouldn't have to be repeated.